web/packages/shared/components/Notification/types.ts

```diff
@@ -22,4 +24,10 @@ export interface NotificationItem {
 
 export type NotificationItemContent =
   | string
-  | { title: string; description: string };
+  | ReactNode
+  | NotificationItemObjectContent;
```

Comment:
I have to say I don't like this solution because it replicates the same problem that I mentioned when we were talking about using Markdown to insert the link: it shifts the responsibility of formatting the notification from the place which displays the notification to the place which wants the notification to be displayed.

If we add support for arbitrary HTML tags, if you want to answer the question of "How do all possible variants of notifications may look like, more or less?", you're no longer able to answer this question without grepping through all places in the codebase which add notifications because each one of them can do some custom stuff.

I think a better play here would be to add an optional `link` field to `NotificationItemObjectContent`. `link` could be an object with fields `href` and `text`. This way we support adding links to notifications while still making it possible to open a storybook and see all possible variants of notifications.

We should update the storybook accordingly as well.

Reply:
I see, thanks for the explanation. 
I added `link` and `list` fields to `Notification`. I assume it should be fine?

Reply:
> I added `link` and `list` fields to `Notification`. I assume it should be fine?

Yeah that looks fine. Technically the type allows a notification with only `title` but we can keep it this way for now.